docs/api-guide.md

```diff
@@ -0,0 +1,718 @@
+# Java API Guide
+
+This guide briefly explains how to use Scalar DB Java API.
+
+## Administrative operations
+
+This chapter explains how to execute administrative operations in Scalar DB.
+
+### Get a DistributedTransactionAdmin instance
+
+To execute administrative operations, you first need to get a `DistributedTransactionAdmin` instance.
+The `DistributedTransactionAdmin` instance can be obtained from `TransactionFactory` as follows:
+
+```java
+TransactionFactory transactionFactory = TransactionFactory.create("<configuration file path>");
+DistributedTransactionAdmin admin = transactionFactory.getTransactionAdmin();
+```
+
+Please see [Getting Started](getting-started.md) for the details of the configuration file.
+
+### Create a namespace
+
+Before creating tables, namespaces must be created since a table belongs to one namespace.
+You can create a namespace as follows:
+
+```java
+// Create a namespace "ns"
+admin.createNamespace("ns");
+
+// Create a namespace if not exists
+boolean ifNotExists = true;
+admin.createNamespace("ns", ifNotExists);
```

Comment:
```suggestion
// Create a namespace "ns". It will throw an exception if the namespace already exists
admin.createNamespace("ns");

// Create a namespace only if it does not already exist
boolean ifNotExists = true;
admin.createNamespace("ns", ifNotExists);
```
What do you think?